--- a/src/storage_v1_types.rs
+++ b/src/storage_v1_types.rs
@@ -2070,6 +2070,8 @@
     /// Responses with Content-Type of application/json
     #[serde(rename = "json")]
     Json,
+    #[serde(rename = "media")]
+    Media,
 }
 
 impl std::default::Default for StorageParamsAlt {
@@ -2083,6 +2085,7 @@
         let s = match self {
             StorageParamsAlt::Undefined => write!(f, "undefined"),
             StorageParamsAlt::Json => write!(f, "json"),
+            StorageParamsAlt::Media => write!(f, "media"),
         };
         Ok(())
     }
