--- a/src/drive_v3_types.rs
+++ b/src/drive_v3_types.rs
@@ -1706,6 +1706,8 @@
     /// Responses with Content-Type of application/json
     #[serde(rename = "json")]
     Json,
+    #[serde(rename = "media")]
+    Media,
 }
 
 impl std::default::Default for DriveParamsAlt {
@@ -1719,6 +1721,7 @@
         let s = match self {
             DriveParamsAlt::Undefined => write!(f, "undefined"),
             DriveParamsAlt::Json => write!(f, "json"),
+            DriveParamsAlt::Media => write!(f, "media"),
         };
         Ok(())
     }
